lib/polyfill.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
This file is no longer needed and has been deleted, but I updated the copyright notice in the other new files that this pull request creates.